plugin/adsk/plugin/adskMaterialCommands.h

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#pragma once
+
+#ifndef ADSK_MAYA_MATERIAL_COMMANDS_H
+#define ADSK_MAYA_MATERIAL_COMMANDS_H
+
+#include <mayaUsd/base/api.h>
+#include <mayaUsd/mayaUsd.h>
+#include <mayaUsd/undo/OpUndoItemList.h>
+
+#include <maya/MPxCommand.h>
+#include <ufe/path.h>
+#include <ufe/undoableCommand.h>
+
+namespace MAYAUSD_NS_DEF {
+
+//------------------------------------------------------------------------------
+// GetMaterialXMaterialsCommand
+//------------------------------------------------------------------------------
+
+struct MxShaderMenuEntry
```

Comment:
I don't think this struct is used. Can you recheck and see it it can be removed?